tests/unit/test_eventtype_parser.py

```diff
@@ -0,0 +1,57 @@
+import pytest
+from unittest.mock import patch
+from pytest_splunk_addon.standard_lib.addon_parser.eventtype_parser import EventTypeParser
+
+
+def test_eventtypes_can_be_parsed_and_extracted(parser):
+    assert parser.eventtypes, "eventypes can not be called or does not return any value"
+
+
+def test_eventtypes_can_be_parsed_and_returned(parser):
+    expected_outputs = [{"stanza": x} for x in ["fake_splunkd", "fake_for_tags_positive"]]
+    for i, event in enumerate(parser.get_eventtypes()):
+        assert event == expected_outputs[i], "expeceted event {} not found".format(expected_outputs[i])
+
+
+def test_get_eventtypes_calls_app_get_config(parser):
+    with patch.object(parser.app, "eventtypes_conf") as mock:
+        for _ in parser.get_eventtypes():
+            pass
+        mock.assert_called_once()
```

Comment:
If you make FakeApp a mock:
```suggestion
    for _ in parser.get_eventtypes():
        pass
    parser.app.eventtypes_conf.assert_called_once()
```